plaso/parsers/onedrive.py

```diff
@@ -0,0 +1,467 @@
+# -*- coding: utf-8 -*-
+"""Parser for OneDrive Log (ODL/ODLGZ) files.
+
+Reference: https://forensics.wiki/microsoft_onedrive/
+"""
+
+import base64
+import io
+import json
+import os
+import re
+import zlib
+
+import pycaes
+
+from cryptography.hazmat.primitives import padding
+
+from dfdatetime import posix_time
+
+from dfvfs.helpers import text_file
+from dfvfs.resolver import resolver as path_spec_resolver
+from dfvfs.path import factory as path_spec_factory
+
+from plaso.containers import events
+from plaso.lib import dtfabric_helper
+from plaso.lib import errors
+from plaso.lib import specification
+from plaso.parsers import interface
+from plaso.parsers import manager
+
+
+class OneDriveLogEvent(events.EventData):
+  """OneDrive log event data.
+
+  Attributes:
+    code_file_name (str): code file name
```

Comment:
nit: add trailing dot, like for recorded_time

Reply:
Done